modules/installation-configuration-parameters.adoc

```diff
@@ -1644,6 +1644,55 @@ Optional VMware vSphere machine pool configuration parameters are described in t
 |Integer
 |====
 
+[id="installation-configuration-parameters-zone-region-vsphere_{context}"]
+== VMware vSphere zone and region enablement configuration parameters
+
+The following table describes VMware vSphere zone and region enablement configuration parameters:
+
+.VMware vSphere zone and region enablement parameters
+[cols=".^2,.^3,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`vsphere`
+|Parameter covers two purposes in the context of a vSphere configuration: (1) Describes your account on the cloud platform that hosts your cluster, and you use the parameter to customize the platform; (2) The parameter is optional when providing additional configuration settings for the machine pool parameters associated with compute and control plane machines.
+
+a|String
+
+[NOTE]
+====
+You can only specify one vCenter server for your {product-title} cluster.
+====
+
+|`vcenters`
+| The fully-qualified hostname or IP address for each listed vCenter server.
+| String
+
+|`datacenters`
+|Defines the multiple datacenters that operate in a single VMware vCenter.
```

Comment:
Something like this instead:
Defines the datacenters where the OCP virtual machines will operate. That list of datacenters **must** match the list of datacenters used in failuredomains.

Reply:
Thank you. I updated with some slight changes.